src/components/events/EventCard/index.tsx

```diff
@@ -69,16 +71,19 @@ const EventCard = ({
         />
       )}
 
-      <button
-        type="button"
+      <Link
+        href={`${config.eventsRoute}/${uuid}`}
         data-community={community}
         className={`${styles.container} ${borderless ? '' : styles.bordered} ${className || ''}`}
-        onClick={() => setExpanded(true)}
+        onClick={e => {
+          e.preventDefault();
+          setExpanded(true);
+        }}
       >
```

Comment:
seeking to learn: what does this link do? Is it a fallback where there's no modal

Reply:
if you middle click or right click it, it doesn't call `onClick` so it behaves like a normal link, so you can open it in a new tab. if you click it normally it opens the modal